src/components/starlight/Footer.astro

```diff
@@ -0,0 +1,11 @@
+---
+import Pagination from './Pagination.astro';
+/* import LastUpdated from '@astrojs/starlight/components/LastUpdated.astro'; */
+import EditLink from './EditLink.astro';
+import References from '../References.astro';
+import type { Props } from '@astrojs/starlight/props';
+---
+<EditLink { ...Astro.props} />
+<!-- <LastUpdated { ...Astro.props} /> -->
```

Comment:
```suggestion
```

Reply:
Sounds good! I wanted to add it in case we wanted to add it for future development. But it can be simply added late if we decide to add it.